people_tool/static/people_tool/js/controllers/CoursesController.js

```diff
@@ -108,8 +106,8 @@
             }
             return 'Untitled Course';
         };
-        $scope.getProfileRoleTypeCd = function(profile) {
-            return profile ? profile.role_type_cd : '';
+        $scope.getProfileIdType = function(profile) {
```

Comment:
Looks like we can completely remove this, I can't see anywhere it's used in this app (it came from the course_info app and may not have been used in this one).